obspy/clients/fdsn/routers/__init__.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+obspy.clients.fdsn.routers - FDSN Federated catalog webservice client for ObsPy
+======================================================
+The obspy.clients.fdsn package contains a client to access web servers that
+implement the FDSN web service definitions (https://www.fdsn.org/webservices/).
+The holdings of many of these services are indexed by the FedCatalog service
+ (https://service.iris.edu/irisws/fedcatalog/1/). Users searching for waveforms
+ or metadata can then query the FedCatalog service to learn which provider
+ holds which data. Furthermore, the results from a FedCatalog query are easily
+ turned into POST requests for each service.
+
+ This FederatedClient first queries the FedCatalog service to determine where
+ the data of interest reside. It then queries the individual web services from
+ each provider using the FDSN Client routines to retrieve the resulting data.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+    Celso G Reyes, 2017
+    IRIS-DMC
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+
+Basic Usage
+-----------
+
+The first step is always to initialize a client object.
+
+>>> from obspy.clients.fdsn import FederatedClient
+>>> client = FederatedClient()
+
+Retrieving Station Metadata
+---------------------------
+
+Submitting a GET request to the federated catalog service. The service
+recognizes parameters that are normally accepted by the station web service.
+
+>>> inv = client.get_stations(station="AN*", channel="BHZ", level="station")
+
+
+Retrieving Waveform Metadata
+---------------------------
+Submitting a GET request to the federated catalog service. The service
```

Comment:
A lot of times I'm seeing these notes about doing `GET` or `POST` requests (FDSN API front page, fdsn/routers API front page etc)... this should be removed from the docs I think, that's a low level implementation aspect that's not interesting for the user.